src/olympia/conf/dev/settings.py

```diff
@@ -261,7 +261,19 @@
         'redirect_url': 'https://amo.dev.mozaws.net/fxa-authenticate',
         'scope': 'profile',
     },
+    'development': {
```

Comment:
I dislike that the server is called `addons-dev`. But node uses `development` so that's what I was thinking. They are quite similar though.

I find `local_development` a bit verbose, I'll probably switch it to just `local`.